use-location.js

```diff
@@ -6,7 +6,7 @@ import { useEffect, useRef, useState, useCallback } from "./react-deps.js";
 const eventPopstate = "popstate";
 const eventPushState = "pushState";
 const eventReplaceState = "replaceState";
-export const events = [eventPopstate, eventPushState, eventReplaceState];
```

Comment:
I also remove this export statement because it isn't being used by any other file

Reply:
Well, this is intended, although not documented. Some apps use the name of the events in order to subscribe to their own location changes. I'm thinking about removing it in the next major release though.

Reply:
Ah ok. Thanks for clarifying.